toolchain/check/node_id_traversal.cpp

```diff
@@ -104,22 +110,45 @@ static auto IsEndOfDeferredDefinitionScope(Parse::NodeKind kind) -> bool {
   }
 }
 
+static auto FinishAndPopDeferredDefinitionScope(Context& context) -> void {
+  for (auto& thunk :
+       context.deferred_definition_scope_stack().PeekPendingThunks()) {
+    BuildThunkDefinition(context, std::move(thunk));
+  }
+
+  context.deferred_definition_scope_stack().Pop();
+}
```

Comment:
Maybe this'll sound odd of me, but this is a long function name, and with no function comments plus a single caller, it might actually be easier to read inline

Reply:
Inlined.